src/sentry/south_migrations/0338_backfill_feature_adoption_first_project.py

```diff
@@ -0,0 +1,901 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            # Explicitly resume the transaction because
+            # South is going to try and roll it back, but when
+            # it can't find one, it'll error itself, masking
+            # the actual exception being raised
+            #
+            # See https://github.com/getsentry/sentry/issues/5035
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+
+    def _forwards(self, orm):
+        from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+        Organization = orm['sentry.Organization']
+        OrganizationOnboardingTask = orm['sentry.OrganizationOnboardingTask']
+        FeatureAdoption = orm['sentry.FeatureAdoption']
+
+        queryset = Organization.objects.all()
+        for org in RangeQuerySetWrapperWithProgressBar(queryset):
```

Comment:
i don't think you need to involve the organization model at all. i think this all could be refactored to something like this, right?

```
FEATURE_MAP = {1: 60, 2: 40, 3: 61}
queryset = OrganizationOnboardingTask.objects.filter(task__in=[1, 2, 3], status=1)
for oo_task in RangeQuerySetWrapperWithProgressBar(queryset):
    try:
        with transaction.atomic():
            FeatureAdoption.objects.create(
                feature_id=FEATURE_MAP[oo_task.task], 
                organization_id=oo_task.organization_id,
                complete=True,
            )
    except IntegrityError:
        pass
```